src/translations/main_ru.ts

```diff
@@ -0,0 +1,811 @@
+    </message>
+    <message>
+        <source>Make the system tray icon</source>
+        <translation>Сделать иконку в системном трее</translation>
+    </message>
+    <message>
+        <source>Check for new updates when Birdtray starts</source>
+        <translation>Проверка наличия обновлений при запуске Birdtray</translation>
+    </message>
+    <message>
+        <source>Check for new updates on startup</source>
+        <translation>Проверка наличия обновлений при запуске</translation>
+    </message>
+    <message>
+        <source>Check for a new Birdtray version</source>
+        <translation>Проверка новой версии Birdtray</translation>
+    </message>
+    <message>
+        <source>Check now</source>
+        <translation>Проверить сейчас</translation>
+    </message>
+    <message>
+        <source>About</source>
+        <translation>О программе</translation>
+    </message>
+    <message>
+        <source>Version check failed</source>
+        <translation>Проверка версии не удалась</translation>
+    </message>
+    <message>
+        <source>Failed to check for a new Birdtray version:
+</source>
+        <translation>Не удалось проверить наличие новой версии Birdtray:
+</translation>
+    </message>
+    <message>
+        <source>Checking...</source>
+        <translation>Проверка...</translation>
+    </message>
+    <message>
+        <source>Choose the new icon</source>
+        <translation>Выберите новую иконку</translation>
+    </message>
+    <message>
+        <source>Images (*.png *.svg *.svgz)</source>
+        <translation>Изображение (*.png *.svg *.svgz)</translation>
+    </message>
+    <message>
+        <source>Invalid icon</source>
+        <translation>Неправильная иконка</translation>
+    </message>
+    <message>
+        <source>Could not load the icon from this file.</source>
+        <translation>Не загружается иконка из файла.</translation>
+    </message>
+    <message>
+        <source>Thunderbird command line:</source>
+        <translation>Командная строка Thunderbird:</translation>
+    </message>
+    <message>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the full command-line (with arguments) which will be used to start Thunderbird. Arguments are space-separated, but spaces in quotes are allowed, i.e. something like &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;C:\Program Files\tb.exe&amp;quot; --profile test&lt;/span&gt; will work.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Это полная командная строка (с аргументами), которая будет использоваться для запуска Thunderbird. Аргументы разделены пробелами, но допускаются пробелы в кавычках, т.е.&lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;C:\Program Files\tb.exe&amp;quot; --profile test&lt;/span&gt; будет работать.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```

Comment:
```suggestion
        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Это полная командная строка (с аргументами), которая будет использоваться для запуска Thunderbird. Аргументы разделены пробелами, но допускаются пробелы в кавычках, т.е. &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;C:\Program Files\tb.exe&amp;quot; --profile test&lt;/span&gt; будет работать.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```
I think we want a space there.